app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -19,6 +19,8 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     DEFAULT_PORT = 80
     default_value_for :port, DEFAULT_PORT
 
+    has_many :middleware_domains, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :middleware_server_groups, :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
is this something that is interesting to see in the context of a provider? (and not as part of a domain?)
@jdoyleoss had some comments in the past regarding that there might be no much benefit showing datasources per provider and not only per a server, hence I'm wondering if this is not a similar case.